packages/stokhos/src/sacado/kokkos/vector/Stokhos_MP_Vector_MaskTraits_expr_relops_tmpl.hpp

```diff
@@ -61,11 +61,11 @@ namespace Sacado {
 #pragma unroll
 #endif
       if (v2.hasFastAccess(v1.size())) {
-        for(std::size_t i=0; i<v1.size(); ++i)
+        for(auto i=0; i<v1.size(); ++i)
```

Comment:
This may result in signed-unsigned comparison warnings.  The type of `i` here is `int` (deduced from `0`), but `v1.size()` may have type `size_t`.